src/main/java/Wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package Wolox.training.models;
+
+import Wolox.training.exceptions.BookAlreadyOwnedException;
+
+import javax.persistence.*;
+import java.util.Collections;
+import java.util.List;
+
+@Entity(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String birthDate;
```

Comment:
The birhdate type should be `LocalDate`
```suggestion
    private LocalDate birthDate;
```